epoxy-paging/src/main/java/com/airbnb/epoxy/paging/PagingEpoxyController.java

```diff
@@ -177,6 +177,24 @@ public void setList(@Nullable PagedList<T> list) {
     updatePagedListSnapshot();
   }
 
+  /**
+     * Returns the list currently being displayed by the EpoxyController.
+     *
+     * @return The list currently being displayed.
+     */
+  public List<T> getCurrentList() {
+    return this.list == null ? Collections.<T>emptyList() : this.list;
```

Comment:
`list` is never null, you can return it directly

Reply:
null check here is unnecessary too.